src/practices/JavaScript/DependenciesVersionMajorLevel.ts

```diff
@@ -0,0 +1,25 @@
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+import { evaluateBySemverLevel, SemverVersion } from '../../detectors/utils';
+import { PracticeEvaluationResult, PracticeImpact, ProgrammingLanguage } from '../../model';
+import { DxPractice } from '../DxPracticeDecorator';
+import { IPractice } from '../IPractice';
+
+@DxPractice({
+  id: 'LanguageIndependent.DependenciesVersionMajorLevel',
+  name: 'Update Dependencies of Major Level',
+  impact: PracticeImpact.small,
+  suggestion: 'Keep the dependencies updated to eliminate security concerns and compatibility issues. Use, for example, Renovate Bot.',
```

Comment:
The suggestion text should say something about new features instead of security fixes. For security fixes are patch versions there.